easybuild/easyconfigs/m/muMerge/muMerge-1.1.0-foss-2022a.eb

```diff
@@ -0,0 +1,43 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'muMerge'
+version = '1.1.0'
+
+homepage = 'https://github.com/Dowell-Lab/mumerge'
+description = """muMerge is a tool for combining bed regions from multiple bed files that overlap."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [('binutils', '2.38')]
```

Comment:
Hmm, you would only need this for a `GCCcore` toolchain?

Reply:
Thanks for finding that, I removed the `binutils`.